data/data/ibmcloud/network/vpc/security-groups.tf

```diff
@@ -17,7 +17,7 @@ locals {
 ############################################
 
 resource "ibm_is_security_group" "cluster_wide" {
-  name           = "${local.prefix}-security-group-cluster-wide"
+  name           = "${local.prefix}-sg-cluster-wide"
```

Comment:
The var names are clear and descriptive. IMO, we should leave them be and not shorten them.